requirements/torch_cu118_linux.txt

```diff
@@ -1,8 +1,5 @@
 #pip install torch torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/cu117
 --extra-index-url https://download.pytorch.org/whl/cu118
-cython
-wheel
```

Comment:
is this needed to build our binaries for pypi?

Reply:
These are a part of common.txt already